OpenShift/manifests/README.md

```diff
@@ -52,27 +52,36 @@ There are single-user and multi-user options to deploy Kubeflow. Choose one that
 
 * Single-user
 
-  Use this [KfDef configuration](./kfctl_tekton_openshift_minimal.v1.1.0.yaml) file to deploy the minimal required components for single-user Kubeflow with Tekton pipeline.
+  Use this [KfDef configuration](./kfctl_openshift_tekton.v1.1.0.yaml) file to deploy the minimal required components for single-user Kubeflow with Tekton pipeline.
```

Comment:
```suggestion
  Use this [KfDef configuration](./kfctl_openshift_tekton.v1.1.0.yaml) file to deploy Kubeflow with Kubeflow Pipelines-Tekton combination. Please note that this manifest is derived from OpenDataHub manifest for Kubeflow, and we have customized Kubeflow Pipelines with Tekton backend.
```

Reply:
Additionally is it version 1.1? Should be KF 1.0?

Reply:
No, this is based on Kubeflow 1.1.0. Only copied the relative openshift-scc and openshift layers for several applications from odh. So all are in Kustomize v3.

Reply:
hmmm - so KF 1.1 can run on OpenShift?

Reply:
Did not try KFP. But for Katib, Seldon etc I am using their manifests in v1.1-branch.